index.js

```diff
@@ -26,6 +28,10 @@ const action = (key) => {
 	if (key.name === 'down')      return 'down'
 	if (key.name === 'right')     return 'right'
 	if (key.name === 'left')      return 'left'
+	if (key.name === 'k')         return 'up'
```

Comment:
I won't be able to type a `k` letter into a prompt, right? We need a mode switch to make it work like vim.

Reply:
Yeah, I believe so.  I wasn't sure the scope of this project as I've only used it in cli-minesweeper and keys aren't really needed for that.  I'll definitely put it behind a flag that's a much better idea.

Reply:
I think even with the flag, it should support input of regular keys.